docs/pages/intro.md

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Introduction'
+weight: 20
+---
+
+[Glee](https://github.com/asyncapi/glee) is a spec-first framework that helps you build server-side applications. It leverages the AsyncAPI specification to make you more productive:
+
+- Glee ensures your code and AsyncAPI definition are on par, eliminating the problem of outdated documentation. Glee takes care of this automatically for you.
```

Comment:
For intro, I think we should add more details on how the code and spec are in sync and why you have to change the spec first in order to build anything in glee.

Reply:
By having both the code and the AsyncAPI definition in sync, you can ensure that the API documentation is always up to date, accurate, and reflects the current state of the application.